tests/test_cmt.py

```diff
@@ -1,8 +1,89 @@
 """Testing functions for the CMT services"""
 
+import strax
 import straxen
+import utilix
 import numpy as np
+from warnings import warn
+from .test_basics import test_run_id_1T
+from .test_plugins import test_run_id_nT
+from straxen.common import aux_repo
 
+def test_connect_to_db():
+    """
+    Test connection to db
+    """
+    if not straxen.utilix_is_configured():
+        warn('Cannot do test becaus  '
+             'no have access to the database.')
+        return
+
+    username=None
+    password=None
+    mongo_url=None
+    is_nt=True
+    mongo_kwargs = {'url': mongo_url,
+                        'user': username,
+                        'password': password,
+                        'database': 'corrections'}
+    corrections_collection = utilix.rundb.xent_collection(**mongo_kwargs)
+    client = corrections_collection.database.client     
+    cmt = strax.CorrectionsInterface(client, database_name='corrections')
+    df = cmt.read('global')
+    mes = 'Return empty dataframe when reading DB. Please check'
+    assert not df.empty, mes
+
+def test_1T_elife():
+    """
+    Test elife from CMT DB against historical data(aux file)
+    """
+    if not straxen.utilix_is_configured():
+        warn('Cannot do test becaus  '
+             'no have access to the database.')
+        return
+
+    elife_conf = ('elife', 'ONLINE', False)

```

Comment:
I have the impression, that this will become difficult in the long run. Maybe we should consider to add to each new correction a "test" column which contains just a static value.

Reply:
we are using a lot of aux/pax fails in testing so don't understand why this is an issue